package.json

```diff
@@ -62,7 +67,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged --pattern 'packages/**/src/**/*' && npx tslint -c tslint.json --project ."
+      "pre-commit": "pretty-quick --staged --pattern 'packages/**/src/**/*' && npx eslint . --ext .ts"
```

Comment:
Shall we add `js` and `json` in `ext` as well, in case of any js script/config files like `eslint.json` or `xxx.config.js`

Reply:
I believe it won't help to check against json files.. cause syntax and type checking won't be required in case of json.. js can be added... created a spike ticket for having js as extension.. Tried locally to run.. but the command  npx eslint . --ext .ts,.js kept running for last 30mins..

Reply:
Good to know that, thanks.